libs/designsystem/card/src/card.component.spec.ts

```diff
@@ -45,36 +49,75 @@ describe('CardComponent', () => {
     });
   });
 
-  describe('with mode attribute', () => {
+  describe('with flat variant attribute', () => {
+    let cardElement: HTMLElement;
+
     beforeEach(() => {
-      spectator = createHost('<kirby-card mode="flat"></kirby-card>');
+      spectator = createHost('<kirby-card variant="flat"></kirby-card>');
+      cardElement = spectator.queryHost('kirby-card');
```

Comment:
Consider using the spectator element property for similar functionality
```suggestion
      cardElement = spectator.element;
```